packages/react/src/host/RemoteComponent.tsx

```diff
@@ -69,11 +70,11 @@ export const RemoteComponent = memo(function RemoteComponent({
   const {children} = attached;
 
   if (children.length === 0) {
-    return <Implementation {...props} />;
+    return <Implementation {...otherProps} {...props} />;
   }
 
   return (
-    <Implementation {...props}>
+    <Implementation {...otherProps} {...props}>
```

Comment:
We don't want `props` to be overridden. So, I put `otherProps` in front. Also, there is no need to do the same for Fragment and Text.